docs/user/learner.rst

```diff
@@ -74,10 +93,30 @@ If you are looking for a specific subject, topic, or term, use the **Search** fe
 .. image:: img/search.png
 	:alt: search page
 
+Exercises
+~~~~~~~~~
+
+Kolibri **Exercises** can require you to do different things: fill in a missing number, write a formula, choose one of the available options, etc. Each correct answer gets you a checkmark, and majority of exercises require 5 correct answers in a row to be completed. Some exercises can offer one or more hints, to help you solving the problem. 
+
+Independent of the required action (writing an answer yourself or choosing one of the options), these are the steps to follow.
+
+#. **Read the question carefully**.
+#. When you are ready to submit the answer, click the **Check answer** button.
+
+   * If the answer is correct and a checkmark appears, click the **Next question** button to proceed.
+   * If the answer is incorrect, click the **Get a hint** button, read the suggestions, and try to answer again.
+
+#. Once you have achieved 5 correct answers in a row, click the **Next item** button, to continue learning with the rest of the material in that topic.
```

Comment:
I believe this is only true if that was the setting defined by the content curator. @rtibbles?